### PR TITLE
Fix `mark-unread` button injection

### LINKED ISSUE #2335: `Mark as unread` button is missing
SSIA

 - `chromium` Version 76.0.3809.100 (Official Build) Arch Linux (64-bit)
 - `refined-github` 19.8.14

2019-08-15-095955_3840x2160_scrot

___
The feature is very important for me, i use it every day, please fix asap :pray:

### PR DESCRIPTION
**Pull request by sharkykh**
Fixes #2335

`.discussion-sidebar` is no longer available.

**Test**:
Open any discussion, look for the "Mark as unread" button

### CODE REVIEW

**Comment by sharkykh** [comment]
Cool solution.

**Comment by fregante** [comment]
> Cool solution.

A bit wordy but it should be flexible and more specific; it should also work for the discussion title (`parse-backticks` has the same issue there)

**Comment by ilyam8** [comment]
please do a patch release, the bug is very annoying, it destroys my workflow 😢
